language/it/info_acp_advancedpolls.php

```diff
@@ -51,9 +53,24 @@
 	'AP_ACT_VOTERS_SHOW_EXPLAIN'	=> 'Attiva l’opzione per mostrare i votanti per ogni risposta del sondaggio.',
 	'AP_ACT_VOTERS_LIMIT'			=> 'Attiva limite per votanti',
 	'AP_ACT_VOTERS_LIMIT_EXPLAIN'	=> 'Attiva l’opzione per limitare il voto a chi abbia prima risposto al topic.',
-
+	'AP_ACT_POLL_NO_VOTE'			=> 'Attiva astensione',
+	'AP_ACT_POLL_NO_VOTE_EXPLAIN'	=> 'Cambia il link standard “Mostra risultati” con “Mi astengo” che non permetterà di votare dopo aver visto i risultati (a meno che non sia attiva l’opzione di cambio voto).',
+	'AP_ACT_SHOW_ORDERED'			=> 'Attiva mostra ordinate',
+	'AP_ACT_SHOW_ORDERED_EXPLAIN'	=> 'Attiva l’opzione per la scelta di visualizzazione dei risultati in ordine decrescente per voti ricevuti (la più votata per prima).',
+	'AP_ACT_POLL_SCORING'			=> 'Attiva punteggi sondaggio',
+	'AP_ACT_POLL_SCORING_EXPLAIN'	=> 'Attiva l’opzione per assegnare punteggi differenti alle opzioni di voto.',
+	'AP_ACT_INCREMENTAL_VOTES'		=> 'Attiva voto incrementale',
+	'AP_ACT_INCREMENTAL_VOTES_EXPLAIN'	=> 'Attiva l’opzione per votare in maniera incrementale, fintanto che non sia estinta la propria possibilità di voto.',
+	'AP_ACT_CLOSED_VOTING'			=> 'Attiva voto chiuso',
+	'AP_ACT_CLOSED_VOTING_EXPLAIN'	=> 'Attiva l’opzione per permettere il voto in sondaggi aperti in topic chiusi.',
+	'AP_ACT_POLL_END'				=> 'Attiva termine voto',
+	'AP_ACT_POLL_END_EXPLAIN'		=> 'Attiva l’opzione per specificate la data e/o l’ora di fine sondaggio, invece di specificarne la durata a partire dall’inizio del sondaggio.',
+	'AP_ACT_POLL_NOTIFICATIONS'				=> 'Attiva notifiche sondaggio',
+	'AP_ACT_POLL_NOTIFICATIONS_EXPLAIN'		=> 'Attiva l’invio di notifiche ai votanti alla scadenza di una votazione con voti nascosti per cui sono visibili i risultati.',
+	
```

Comment:
----------------------------------------------------------------------
 70 | ERROR | [x] Whitespace found at end of line
    |       |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
----------------------------------------------------------------------